CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Changes since the last non-beta release.
 
 _Please add entries here for your pull requests that are not yet released._
 
+- Prevent roots from being re-created when using React 18
```

Comment:
Always provide a URL link, like [done for Shakapacker](https://raw.githubusercontent.com/shakacode/shakapacker/master/CHANGELOG.md):

```
- Fixed creation of assets:precompile if it is missing [PR 325](https://github.com/shakacode/shakapacker/pull/325) by [ahangarha](https://github.com/ahangarha).
```